certdb/pg/migrations/001_CreateCertificates.sql

```diff
@@ -3,16 +3,25 @@
 
 CREATE TABLE certificates (
   id                serial,
-  serial            bytea NOT NULL PRIMARY KEY,
+  serial            bytea NOT NULL UNIQUE,
   ca_label          bytea NOT NULL,
   status            bytea NOT NULL,
   reason            int,
   expiry            timestamptz,
   revoked_at        timestamptz,
-  pem               bytea NOT NULL
+  pem               bytea NOT NULL,
+  PRIMARY KEY(id, serial)
 );
 
+CREATE TABLE ocsp_responses (
+  id                serial PRIMARY KEY,
+  serial            bytea NOT NULL,
+  body              bytea NOT NULL,
+  expiry            timestamptz,
+  FOREIGN KEY(serial) REFERENCES certificates(serial)
```

Comment:
it's probably better to do the foreign key on `certificate.id`:
`certificate_id integer REFERENCES certificates (id)`